scripts/_common.sh

```diff
@@ -87,8 +87,8 @@ ffmpeg_deps_bookworm=(
 )
 
 case "$debian" in
-	bullseye) ffmpeg_deps=$ffmpeg_deps_bullseye ;;
- 	bookworm) ffmpeg_deps=$ffmpeg_deps_bookworm ;;
+	bullseye) ffmpeg_deps="${ffmpeg_deps_bullseye[*]}" ;;
+ 	bookworm) ffmpeg_deps="${ffmpeg_deps_bookworm[*]}" ;;
```

Comment:
Nuuuuuu

`ffmpeg_deps=("${ffmpeg_deps_bookworm[@]}")`

allows you to keep it as a list

(yeah okay you don't care about this as it's transformed as a string just after)

Reply:
baaaaaaaaaaaaaaaaaaaaaaaaaaaaash /o\